cabi/src/unreal.rs

```diff
@@ -29,14 +29,23 @@ ffi_fn! {
 }
 
 ffi_fn! {
-    unsafe fn symbolic_unreal4_get_context(unreal: *mut SymbolicUnreal4Crash) -> Result<SymbolicStr> {
+    unsafe fn symbolic_unreal4_get_context(unreal: *const SymbolicUnreal4Crash) -> Result<SymbolicStr> {
         let unreal = &*(unreal as *const Unreal4Crash);
 
         let context = unreal.get_context()?;
         Ok(SymbolicStr::from_string(serde_json::to_string(&context)?))
     }
 }
 
+ffi_fn! {
+    unsafe fn symbolic_unreal4_get_logs(unreal: *const SymbolicUnreal4Crash) -> Result<SymbolicStr> {
+        let unreal = &*(unreal as *const Unreal4Crash);
+
+        let context = unreal.get_logs(100)?;
```

Comment:
Discussed offline. Since there is no reason to change this value anytime soon, we can leave it hard coded for now. We will re-evaluate this with the next major release of symbolic.